src/libs/feature-previews-config.js

```diff
@@ -1,3 +1,6 @@
+export const JUPYTERLAB_GCP_FEATURE_ID = 'jupyterlab-gcp'
+export const ENABLE_JUPYTERLAB_ID = 'enableJupyterLabGCP'
```

Comment:
It's a bit odd because it's used in a few components that 'consume' the feature switch. Seems like the most logical place to contain this feature-flag related constant was in the file defining the different feature previews, since it didn't belong to just one IA-specific file.

Reply:
👍  Fair enough.